packages/base/all/vendor-config-onl/src/bin/onl-support

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from shutil import copytree, ignore_patterns, copy
+from onl.platform.current import OnlPlatform
+import errno
+import subprocess
+import os
+import logging
+import tarfile
+
+BASE_ROOT = "/tmp/"
+
+DIR_NAME = "onl-support-" + OnlPlatform().PLATFORM
+DIR_ROOT = BASE_ROOT + DIR_NAME
+COMMAND_ROOT = DIR_ROOT + "/support/"
+TARFILE_NAME = DIR_NAME + ".tar.gz"
+TARFILE = BASE_ROOT + TARFILE_NAME
+
+# Set logger
+logger = logging.getLogger('onl-support')
+logger.setLevel(logging.DEBUG)
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+logger.addHandler(ch)
+
+
+def copy_dir(src_dir):
+    dest = DIR_ROOT + src_dir
+    try:
+        copytree(src_dir, dest, symlinks=False, ignore=ignore_patterns('alternatives', '*.deb', '*.swi'))
+    except OSError as e:
+        if e.errno == errno.ENOTDIR:
```

Comment:
Make sure file exist there:

`if e.errno == errno.ENOTDIR and os.path.exist(src_dir): `